admin/assets/javascripts/global-scripts.js

```diff
@@ -18,7 +18,6 @@ $(function () {
         if (validation) {
           validationStatus = validation()
         }
-        console.log('VALIDATE', validationStatus)
```

Comment:
just a reminder that all server side console.log statements should now be `winston.info|debug|warn|error` :)

Reply:
@GuyHarwood Thanks for the reminder! That was WIP done before winston was in place.

Reply:
@pris54 this particular one is client side, which will still use console.  just server side ones that can rely on winston :)